src/Outsystems.SetupTools/Functions/Install-OSPlatformServiceCenter.ps1

```diff
@@ -90,7 +90,14 @@ function Install-OSPlatformServiceCenter
                 }
                 { ($_  -as [int]) -ge 11}
```

Comment:
I'd change this to 
```
                { ($_  -as [int]) -eq 11}
                {
                    if ((($(([version]$osVersion).Minor) -lt 18) -or ($(([version]$osVersion).Minor) -eq 18 -and $(([version]$osVersion).Build) -lt 1)))
                    {
                        $scInstallerArguments = '-file ServiceCenter.oml -extension OMLProcessor.xif IntegrationStudio.xif PlatformLogs.xif'
                    }
```

Reply:
I don't think we're ever going to release a O12 but it was just for a better legibility. This is not a blocker, we can keep that assumption

Reply:
Isn't line 95 quite similar to line 99? Proposal:
`
 #pseudocode
$scInstallerArguments = '-file ServiceCenter.oml -extension OMLProcessor.xif IntegrationStudio.xif PlatformLogs.xif'
if (version >= 11.18.1)
{
$scInstallerArguments += ' CentralizedPlatformLogs.xif'
}`

Reply:
Good suggestion, changes were made 💪